opsmngr/version_manifest_test.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opsmngr
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/go-test/deep"
+)
+
+func TestVersionManifest_Get(t *testing.T) {
+	client, _, teardown := setup()
+	defer teardown()
+	const version = "4.4.json"
+
+	manifest, _, err := client.VersionManifest.Get(ctx, version)
+	if err != nil {
+		t.Fatalf("VersionManifest.Get returned error: %v", err)
+	}
+
+	if manifest.Updated != 1599868800044 {
+		t.Errorf("Expected: %v, Got:%v", 1599868800044, manifest.Updated)
+	}
+}
```

Comment:
here I cannot use `mux.HandleFunc` because I am directly calling `http.NewRequest` instead of calling `s.Client.NewRequest`. This test downloads all the time the version manifest 4.4 and checks that the updated field is correct 
<img width="547" alt="Screenshot 2020-09-25 at 09 02 12" src="https://user-images.githubusercontent.com/5663078/94236607-d0be1f80-ff0d-11ea-903d-6fe6854a93bf.png">

Reply:
You say the always downloads the 4.4 manifest - where is that manifest defined? What will happen to the test when we go to 5.0/ another version?

Reply:
So the `Get` function downloads the Version Manifest from `https://opsmanager.mongodb.com/static/version_manifest/`. 
For this particular test I am downloading always the version 4.4 from https://opsmanager.mongodb.com/static/version_manifest/4.4.json. When we will have 5.0, since that new version will be available at `https://opsmanager.mongodb.com/static/version_manifest/5.0.json`, the link https://opsmanager.mongodb.com/static/version_manifest/4.4.json will still work and so this test

Reply:
Excellent, thanks for the clarification. SGTM.